jest.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   testEnvironment: 'node',
-  roots: ['<rootDir>/test'],
+  roots: ['<rootDir>/test', '<rootDir>/api', '<rootDir>/utils'],
```

Comment:
Do we want to just put all tests in `/test`? Or should we move the CDK tests into their respective directories and make this `<rootDir>` or `<rootDir>/**` or something?

Reply:
I think in the past @jeffsegal-ccpo has expressed liking having tests "next to" the code they test (iirc)? Maybe it'd be better to move the infra tests (esp since you've got the rest of these tests in place already)?

Reply:
Thanks @kylelaker-ccpo, that has generally been my preference as modules get larger and you have to scroll three screens down to switch between the source and test files. Happy to defer to team consensus on this though. the previous config was 
```
    roots: ['<rootDir>'],
    testMatch: ['**/*.test.ts'],
```